src/backend/aspen/api/views/pango_lineages.py

```diff
@@ -0,0 +1,24 @@
+import sqlalchemy as sa
+from fastapi import APIRouter, Depends
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from aspen.api.deps import get_db
+from aspen.api.schemas.pango_lineages import PangoLineagesResponse
+from aspen.database.models import PangoLineage
+
+router = APIRouter()
+
+
+@router.get("/", response_model=PangoLineagesResponse)
+async def list_pango_lineages(db: AsyncSession = Depends(get_db)):
```

Comment:
We probably want to include the output type here.

Reply:
Yeah, honestly I skipped over that because of the interaction between `response_model` and the view's brief return before the automagic alley-oop to Pydantic seemed weird to document. That would definitely be a benefit of explicitly using the schema response in the code rather than relying on `response_model`.